website/public/js/controllers/notificationCtrl.js

```diff
@@ -154,9 +154,9 @@ habitrpg.controller('NotificationCtrl',
     });
 
     // Quest invitation modal
-    $rootScope.$watch('party.quest.key && !party.quest.active && party.quest.members[user._id] == undefined', function(after, before){
-      if (after == before || after != true) return;
-      $rootScope.openModal('questInvitation');
+    $rootScope.$watch('user.party.quest.RSVPNeeded && (user.party.quest.completed == "" || user.party.quest.completed == undefined)', function(after, before){
```

Comment:
For `(user.party.quest.completed == "" || user.party.quest.completed == undefined)`, is it better to put the undefined case first? Will ` == ""` throw an error or warning if `user.party.quest.completed` does not exist?